src/fengari-web.js

```diff
@@ -128,20 +128,37 @@ const run_lua_script_tag = function(tag) {
 	}
 };
 
-/* watch for new <script type="text/lua"> tags added to document */
+const contentTypeRegexp = /^(.*?\/.*?)([\t ]*;.*)?$/;
+const maybe_run_lua_script_tag = function(tag) {
```

Comment:
kind of a mouthful, also no indication on what are the criteria for the _maybe_

Reply:
Got a suggestion for a new name?

Reply:
`run_lua_mime_script_tag` ?

Reply:
Is that any better? The function checks if the HTML element is a suitable script tag, and if so: runs it.

Reply:
then the _maybe_ is unnecessary: only `lua_script_tag` will be run

Reply:
But it might not be run in some circumstances.
i.e. it's "maybe run the lua code in this tag"

Reply:
what circumstances ?

Reply:
That it's a script tag of a suitable mime type. Perhaps later we'll add extra conditions.